assignment/scripts/aboutMe.js

```diff
@@ -46,14 +60,44 @@
 // assigns the highest value to `mostPets`. There's several possibilities --
 // be sure to think through all the scenarios. 
 // console.log `mostPets` after the conditional has run.
+let mostPets = 0;
+// check if pets = friendsPets
+if (pets === friendsPets){
+    mostPets = pets;
+    console.log(mostPets);
+} else if (pets > friendsPets){
+    mostPets = pets;
+    console.log(mostPets);
+} else {
+    mostPets = friendsPets;
+    console.log(mostPets);
```

Comment:
Since you are logging the same thing each time, you could remove these console logs from the if-blocks and just put one after.